src/main/kotlin/io/github/enyason/predictable/PredictableValidator.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.enyason.io.github.enyason.predictable
+
+import io.github.enyason.predictable.Predictable
+
+fun Predictable.validate() {
+    if (modelId.isEmpty()) throw IllegalArgumentException("Provided an empty model ID")
```

Comment:
Model ID doesn't seem to be a required field for all create prediction usages

Reply:
can we think on how we are going to handle those models? because the endpoint for those models changes as well.

here are examples models:

https://replicate.com/meta/llama-2-70b-chat/api

https://replicate.com/mistralai/mixtral-8x7b-instruct-v0.1/api


I don't think we can do this in this PR. @mayorJAY